src/popup/pages/JwtSign.vue

```diff
@@ -72,6 +72,10 @@ export default defineComponent({
 
         const dataToSign = JSON.parse(payload);
 
+        /**
+         * Sdk should be initialized in order to get a current networkId.
+         */
```

Comment:
No need to use DocBlock here.

```suggestion
        // Sdk should be initialized in order to get a current networkId.
```